apps/dashboard/test/system/jobs_app_test.rb

```diff
@@ -150,18 +167,31 @@ def script_yml(dir)
   test 'creating and showing scripts' do
     Dir.mktmpdir do |dir|
       setup_project(dir)
-      find('[href="/projects/1"]').click
-      click_on 'New Script'
-      find('#script_title').set('the script title')
-      click_on 'Save'
+      setup_script(1)
+
       expected_yml = <<~HEREDOC
         ---
         title: the script title
         form:
         - cluster
+        - auto_scripts
         attributes:
           cluster:
+            widget: select
             label: Cluster
+            options:
+            - oakley
+            - owens
+            help: ''
+            required: false
+          auto_scripts:
+            options:
+            - - my_cool_script.sh
+              - "#{dir}/projects/1/my_cool_script.sh"
+            - - my_cooler_script.bash
+              - "#{dir}/projects/1/my_cooler_script.bash"
+            directory: "#{dir}/projects/1"
+            label: Script
```

Comment:
This is one of the main features of the PR - the fact that we're serializing (writing to a yaml file) the `widget` type as well as the options for a select widget.